feat(serverless): add PN info (#3739)
scaleway/docs-content: faq/serverless-functions.mdx

There are no constraints when changing a function runtime, you simply need to choose the runtime version you want.
Upgrading a runtime is highly recommended in case of deprecation, and for runtimes that have reached end-of-support or end-of-life. See the [functions runtimes lifecycle documentation](/serverless/functions/reference-content/functions-lifecycle/) for more information.

> ## How to configure access to a Private Network ?

Reviewer: ```suggestion
## How do I configure access to a Private Network?
```

> Scaleway Serverless Functions does not support Virtual Private Networks (VPC) for now, it's a feature under development.

Reviewer: ```suggestion
Scaleway Serverless Functions does not currently support Scaleway VPC or Private Networks, though this feature is under development.
```

> If it's possible to add some network restrictions on your resource, here is the list of prefixes used at Scaleway: [https://www.scaleway.com/en/peering/](https://www.scaleway.com/en/peering/). Serverless ressources does not have dedicated and predictible IP address.

Reviewer: ```suggestion
To add network restrictions on your resource, consult the [list of prefixes used at Scaleway](https://www.scaleway.com/en/peering/). Note that Serverless resources do not have dedicated or predictable IP addresses.
```
